core/src/config.ts

```diff
@@ -121,4 +121,8 @@ export type WidgetsConfig = {
 	 * toast widget config
 	 */
 	toast: ToastProps;
+	/**
+	 * collapse widget config
+	 */
+	collapse: any;
```

Comment:
You have added the collapse component at the core-bootstrap level. So I think there should be nothing about the collapse at the core level.